Assets/Scripts/DontDestroy.cs

```diff
@@ -5,13 +5,18 @@
 public class DontDestroy : MonoBehaviour
 {
     public GameManager manager;
+
     private int isFirstDay;
```

Comment:
If it's isFirstDay then you should make it a boolean value, if you're tracking number of days you should use number of days as a variable name instead or dayNumber or something.